### PR TITLE
[ci skip]Chinese annotation

### PR DESCRIPTION
**Pull request by VisualSJ**
CCMap.h
CCNinePatchImageParser.h

### CODE REVIEW

File: cocos/base/CCMap.h [hunk 1]
```diff
@@ -45,12 +45,18 @@
...
- * @warning The element should be `Ref` or its sub-class.
- * @js NA
- * @lua NA
- */
+  * @~english
+  * Similar to std::unordered_map, but it will manage reference count automatically internally.
+  * Which means it will invoke Ref::retain() when adding an element, and invoke Ref::release() when removing an element.
+  * @~chinese 
+  * 类似于std::unordered_map,但是在内部自动管理引用计数。
+  * 这意味着它将调用裁判
```
Comment (natural-law) [inline cocos/base/CCMap.h]: @VisualSJ , The comment in this line is not complete.

File: cocos/base/CCMap.h [hunk 2]
```diff
@@ -65,27 +71,31 @@ class Map
     // Iterators
     // ------------------------------------------
 
-    /** Iterator, can be used to loop the Map. */
+    /** @~english Iterator, can be used to loop the Map.  @~chinese 迭代器,可用于循环地图。*/
```
Comment (natural-law) [inline cocos/base/CCMap.h]: 循环地图？？
应该是遍历映射？遍历 map？

File: cocos/base/CCMap.h [hunk 3]
```diff
@@ -119,9 +130,14 @@ class Map
...
     
-    /** 
+    /**
+     * @~english
      * Destructor.
      * It will release all objects in map.
+     *
+     * @~chinese 
+     * 析构函数。
+     * 释放所有对象地图。
```
Comment (natural-law) [inline cocos/base/CCMap.h]: 释放 map 中的所有对象。

File: cocos/base/CCMap.h [hunk 4]
```diff
@@ -137,7 +153,7 @@ class Map
 #endif
     }
     
-    /** Returns the number of buckets in the Map container. */
+    /** @~english Returns the number of buckets in the Map container.  @~chinese 返回地图容器buckets的数量。 */
```
Comment (natural-law) [inline cocos/base/CCMap.h]: 地图？

File: cocos/base/CCMap.h [hunk 5]
```diff
@@ -157,7 +173,7 @@ class Map
 #endif
     }
     
-    /** Returns the bucket number where the element with key k is located. */
+    /** @~english Returns the bucket number where the element with key k is located.  @~chinese 返回桶元素k与关键所在。 */
```
Comment (natural-law) [inline cocos/base/CCMap.h]: 语句不通

**Comment by VisualSJ** [comment]
@natural-law please review it again...  Thanks

**Comment by VisualSJ** [comment]
@natural-law I have submitted two files to this pr

**Comment by zilongshanren** [comment]
@natural-law  any updates?

File: cocos/base/ccUtils.h [hunk 6]
```diff
@@ -31,66 +31,107 @@ THE SOFTWARE.
...
+ * Examples:
+ * - If "value" is 15, it will return 16.
+ * - If "value" is 16, it will return 16.
+ * - If "value" is 17, it will return 32.
+ *
+ * @~chinese 返回下一个2的n次方数字
+ * 例子:
+ * ——如果“value”是15,它将返回16。
+ * ——如果“value”是16,它将返回16。
+ * ——如果17“value”,它将返回32。
```
Comment (natural-law) [inline cocos/base/ccUtils.h]: 17“value” -> value 是 17

**Comment by natural-law** [comment]
@andyque 
It's ok for merge.

**Pull request merged**